environment.py

```diff
@@ -52,6 +53,38 @@ def env() -> Mapping[str, Optional[str]]:
         #
         # The first catalog listed is the default catalog.
         #
+        # Sources identify Terra Data Repository datasets, snapshots, or
+        # canned staging areas to index.
+        #
+        # The syntax for TDR datasets or snapshots in EBNF is:
+        #
+        # sources = source (',', source )* ;
+        #
+        # source = 'tdr:', Google Cloud project name,
```

Comment:
Combine the grammars so this reads:
```suggestion
        # source = TDR source | canned source
```